### ISSUE TITLE
jsx-wrap-multiline can't handle JSX fragment syntax

### ISSUE THREAD

**Issue by pelotom** [palantir/tslint#3547]
### Bug Report

- __TSLint version__: 5.8
- __TypeScript version__: 2.6.2
- __Running TSLint via__: (pick one) CLI

TypeScript 2.6.2 [added support](https://github.com/Microsoft/TypeScript/pull/19249) for [JSX fragment syntax](https://github.com/facebook/jsx/issues/84). However the `jsx-wrap-multiline` rule doesn't like it. For example, this code should be fine, but it gets flagged:

```ts
const fragment = (
  <>
    <div>
      hi
    </div>
  </>
);
// Error: Multiline JSX elements must be wrapped in parentheses
```

**Comment by ajafff** [comment]
I think you want to report this in https://github.com/palantir/tslint-react instead

**Comment by pelotom** [comment]
Right-o, sorry. It’s already been reported there: https://github.com/palantir/tslint-react/issues/124
